docs/reference/elements/py-config.md

```diff
@@ -246,7 +246,8 @@ The following optional values are supported by `<py-config>`:
 | `packages` | List of Packages | Dependencies on 3rd party OSS packages are specified here. The default value is an empty list. |
 | `fetch` | List of Stuff to fetch | Local Python modules OR resources from the internet are to be specified here using a Fetch Configuration, described below. The default value is an empty list. |
 | `plugins` | List of Plugins | List of Plugins are to be specified here. The default value is an empty list. |
-| `runtimes` | List of Runtimes | List of runtime configurations, described below. The default value contains a single Pyodide based runtime. |
+| `interpreter` | List of Interpreter| List of Interpreter configurations, described below. The default value contains a single Pyodide based interpreter. |
```

Comment:
1. `interpreter` --> `interpreters` since this is a list and thus, the corresponding key will have to be changed to plural as well.

2. `List of Interpreter|` --> `List of Interpreters |`

Reply:
Yeah that's why I mentioned in the previous team meeting, it's a bit confusing that we are using plural and a list, but you can only use a single interpreter.

I'll made the changes for the plural form, but I think multi-interpreters are still far away so I think using a single one is probably a good idea